x-pack/plugins/ml/public/application/trained_models/models_management/expanded_row.tsx

```diff
@@ -125,41 +118,33 @@ export const ExpandedRow: FC<ExpandedRowProps> = ({ item }) => {
     license_level,
   };
 
-  const {
-    services: { share },
-  } = useMlKibana();
-
   useEffect(
-    function updateDeploymentState() {
+    function updateModelItems() {
       (async function () {
-        const { nodes, ...deploymentStats } = stats.deployment_stats ?? {};
-
-        if (!isPopulatedObject(deploymentStats)) return;
+        const deploymentStats = stats.deployment_stats;
 
-        const result = formatToListItems(deploymentStats)!;
+        if (!deploymentStats) return;
 
-        const items: EuiListGroupItemProps[] = await Promise.all(
-          nodes!.map(async (v) => {
-            const nodeObject = Object.values(v.node)[0];
-            const href = await mlLocator!.getUrl({
-              page: ML_PAGES.TRAINED_MODELS_NODES,
-              pageState: {
-                nodeId: nodeObject.name,
-              },
-            });
-            return {
-              label: nodeObject.name,
-              href,
-            };
-          })
-        );
-
-        result.push({
-          title: 'nodes',
-          description: <EuiListGroup size={'s'} gutterSize={'s'} listItems={items} />,
+        const items: AllocatedModel[] = deploymentStats.nodes.map((n) => {
+          const nodeName = Object.values(n.node)[0].name;
+          return {
+            ...deploymentStats,
+            node: {
+              ...pick(n, [
+                'average_inference_time_ms',
+                'inference_count',
+                'routing_state',
+                'last_access',
+                'number_of_pending_requests',
+                'start_time',
+              ]),
+              name: nodeName,
+            } as AllocatedModel['node'],
+            model_id: 'qq',
```

Comment:
Area all model_id 'qq'? If yes I think we should move to a constant. If not I think this should be updated.

Reply:
Thanks for spotting this, I actually need to remove this line

Reply:
Removed in 27b15636418549fe0479f915f8593db77f3a1a90